lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -8,7 +8,8 @@ class JsonApi < Base
 
         def initialize(serializer, options = {})
           super
-          @included = ActiveModel::Serializer::Utils.include_args_to_hash(instance_options[:include])
+          @include_tree = ActiveModel::Serializer::IncludeTree.from_include_args(options[:include])
```

Comment:
do we need to have the fully qualified namespace?

since we are already under the `Serializer` namespace, wouldn't just `IncludeTree` work?